### ISSUE TITLE
Error, wrong classes... (an intransitive permutation group of degree 10+15)

### ISSUE THREAD

**Issue by dimpase** [gap-system/gap#5528]
### Observed behaviour
```
gap> H:=Group([ (1,2,3,4,5), (3,4,5), (6,7,8,9,10), (8,9,10),
(11,12,13,14,15),
>   (13,14,15), (16,17,18,19,20), (18,19,20), (21,22,23,24,25), (23,24,25),
>   (26,27,28), (1,6)(2,7)(3,8)(4,9)(5,10)(16,21)(17,22)(18,23)(19,24)(20,25),
>   (11,16,21)(12,17,22)(13,18,23)(14,19,24)(15,20,25) ]);;
gap> csg := List(ConjugacyClasses(H), Representative);;
Error, wrong classes at /mnt/d/works/ubuntu/gap-4.12.2/lib/clashom.gi:2535
called from
ConjugacyClassesViaRadical( G ) at
/mnt/d/works/ubuntu/gap-4.12.2/lib/clashom.gi:3120 called from
<function "ConjugacyClasses perm group">( <arguments> )
 called from read-eval loop at *stdin*:5
you can 'quit;' to quit to outer loop, or
you can 'return;' to continue
```

### Expected behaviour
it should work

### Copy and paste GAP banner (to tell us about your setup)
```
 ┌───────┐   GAP 4.12.2 of 2022-12-18
 │  GAP  │   https://www.gap-system.org
 └───────┘   Architecture: x86_64-pc-linux-gnu-default64-kv8
 Configuration:  gmp 6.3.0, GASMAN, readline
 Loading the library and packages ...
 Packages:   Alnuth 3.2.1, AtlasRep 2.1.6, AutPGrp 1.11, CRISP 1.4.6, CTblLib 1.3.4, FactInt 1.6.3, FGA 1.4.0, GAPDoc 1.6.6, 
             IRREDSOL 1.4.4, LAGUNA 3.9.5, Polenta 1.3.10, Polycyclic 2.16, PrimGrp 3.4.3, RadiRoot 2.9, ResClasses 4.7.3, 
             SmallGrp 1.5.1, Sophus 1.27, TomLib 1.2.9, TransGrp 3.6.3, utils 0.81
 Try '??help' for help. See also '?copyright', '?cite' and '?authors'
```

it was reported on support by `Sumi Toshio <sumi@artsci.kyushu-u.ac.jp>`

I am able to reproduce this (the banner is from my run)

Same error with `GAP 4.13dev-442-g53703f1 built on 2023-10-15 12:37:29+0100` as well as the latest master branch:
`GAP 4.13dev-446-gc4a3907 built on 2023-11-09 22:27:56+0000`

**Comment by dimpase** [comment]
It's an intransitive group, two orbits (none faithful)
```
gap> StructureDescription(Action(H,o[2]));
"(A5 x A5 x A5) : S3"
gap> StructureDescription(Action(H,o[1]));
"(A5 x A5) : C2"
```

**Comment by dimpase** [comment]
fixed by #5521
